my_enumerable.rb

```diff
@@ -0,0 +1,13 @@
+module Enumerable
+  def all(list)
```

Comment:
A method that returns a bool should have `?` at the end so it should be defined as `all?` not `all`.
`all?` should return `true` if all elements meet a specified condition and false otherwise. For example if the list = [1, 2, 3, 4], then `list.all? {|e| e < 5}` returns `true`.